lib/Controller/StackController.php

```diff
@@ -30,12 +30,13 @@
 use OCP\AppFramework\Controller;
 
 class StackController extends Controller {
-	private $userId;
-	private $stackService;
-	public function __construct($appName, IRequest $request, StackService $stackService, $userId) {
+	public function __construct(
+		$appName,
```

Comment:
Maybe we can directly type hint this one as well:

```suggestion
		string $appName,
```